core/src/test/scala/org/apache/spark/rdd/PairRDDFunctionsSuite.scala

```diff
@@ -30,6 +30,7 @@ import org.apache.hadoop.mapreduce.{JobContext => NewJobContext,
   OutputCommitter => NewOutputCommitter, OutputFormat => NewOutputFormat,
   RecordWriter => NewRecordWriter, TaskAttemptContext => NewTaskAttempContext}
 import org.apache.hadoop.util.Progressable
+import org.scalatest.Assertions
```

Comment:
do you really need this import?  other tests are already calling `assert` and `===` ...

Reply:
If I remove this import, It will give me an error

Reply:
oh right, i see where you are using it now, sorry